pkg/testutil/plugin.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/krew/pkg/constants"
+	"sigs.k8s.io/krew/pkg/index"
+)
+
+var (
+	defaultName             = "test-plugin"
+	defaultShortDescription = `in-memory test plugin object`
+	defaultVersion          = `v1.0.0-test.1`
+	defaultMeta             = metav1.TypeMeta{
+		APIVersion: constants.CurrentAPIVersion,
+		Kind:       constants.PluginKind,
+	}
+
+	defaultPlatformURI    = "http://example.com/"
+	defaultPlatformSHA256 = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef"
+	defaultPlatformOSArch = map[string]string{
+		"os":   "linux",
+		"arch": "amd64"}
```

Comment:
super-nit: that closing brace looks a bit awkward.

Reply:
this syntax is fine :D